### ISSUE TITLE
5G connection type check error

### ISSUE THREAD

**Issue by polmum** [mopub/mopub-ios-sdk#372]
CTRadioAccessTechnologyNR and CTRadioAccessTechnologyNRNSA checks crash (or fail to compile) under certain conditions.

- [x] I am submitting a bug report for existing functionality
- [x] I visited https://developers.mopub.com/ and found no answer
- [x] I checked https://twittercommunity.com/c/advertiser-api/mopub to make sure that this issue has not already been filed
- [x] I checked to make sure that this issue has not already been filed

#### Steps to reproduce the behavior:
An iPhone using 5G with iOS 14.0.
Or compiling with Xcode 12.0 and iOS 14.0

#### Expected behavior:
Everything works correctly

#### Observed behavior:
Code should crash when checking [here](https://github.com/mopub/mopub-ios-sdk/blob/484b1fff1a136023e1658ee9b5a64b89a504cd7b/MoPubSDK/Internal/CellularService.swift#L70).
If using Xcode 12.0, compilation fails with `Use of undeclared identifier CTRadioAccessTechnologyNR`

#### Evidence:
Apple marked CTRadioAccessTechnologyNR and CTRadioAccessTechnologyNRNSA incorrectly as available on iOS 14.0+, as they are only available on iOS 14.1+. They have corrected their docs to reflect [this](https://developer.apple.com/documentation/coretelephony/ctradioaccesstechnologynr), and also the upcoming iOS 14.5 beta fixes the availability declaration:
```
CORETELEPHONY_EXTERN NSString * const CTRadioAccessTechnologyNRNSA         API_AVAILABLE(ios(14.1)) API_UNAVAILABLE(macos);
CORETELEPHONY_EXTERN NSString * const CTRadioAccessTechnologyNR            API_AVAILABLE(ios(14.1)) API_UNAVAILABLE(macos);
```

Solution:
MoPub's code should use a `if (@available(iOS 14.1, *))` check instead.

**Comment by chauduyphanvu** [comment]
@polmum, thanks for the report. We're aware of that and have resolved it internally. The fix will be deployed in the upcoming MoPub SDK release.

Meanwhile, make sure that you are using Xcode 12.1 at the minimum so that the build succeeds. Thanks!

Vu Chau
MoPub Team

**Comment by polmum** [comment]
Hey!
I wanted to provide more feedback: with the Xcode 12.5 betas, MoPub is failing to compile.

**Comment by chauduyphanvu** [comment]
@polmum - thanks for the feedback. I have relayed it to the rest of the team. We will discuss what we can do here.

**Comment by chauduyphanvu** [comment]
@polmum - this issue has been addressed in v5.16.2: https://github.com/mopub/mopub-ios-sdk/releases/tag/5.16.2. Please update and let us know if you continue seeing the error. Thanks!
